pkg/controller/configmap/config.go

```diff
@@ -56,6 +56,7 @@ func (adaptor *ConfigMapK8s) FillDefaults(configmap *corev1.ConfigMap, spec *con
 		log.Error(err, "failed to load ConfigMap")
 		return err
 	}
+	appendErrorIfNotNil(&errs, fillDefault(cfg, "coe", "adaptor", "kubernetes", true))
```

Comment:
fill default coe adaptor value since this option is hidden in configmap, the OC already did it in code line#82